fix(regular_expression): Improve RegExp `to_string()` results

leaysgur opened the pull request:
Hopefully fixes #5487 

- `/^([a-zA-Z]+:)?[\p{L}0-9@/.\-_\\]+$/u`
- https://github.com/react-native-community/cli/blob/e3bc0ddcba67cdd6b7fd6c04b5a8a87931e9d086/packages/cli-tools/src/launchEditor.ts#L53-L57

Boshen commented:
<img width="943" alt="image" src="https://github.com/user-attachments/assets/879c771b-4aa3-4f37-a994-ff821427c309">

You can trigger the CI like this.

leaysgur committed:
fix(regular_expression): Improve RegExp `to_string()` results (#5635)

Hopefully fixes #5487

- `/^([a-zA-Z]+:)?[\p{L}0-9@/.\-_\\]+$/u`
- https://github.com/react-native-community/cli/blob/e3bc0ddcba67cdd6b7fd6c04b5a8a87931e9d086/packages/cli-tools/src/launchEditor.ts#L53-L57

leaysgur commented:
Thanks! I'll try next time.

(That UI only appears when latest one is failed...?)

![image](https://github.com/user-attachments/assets/931bb32a-6d61-4dad-9b2e-f937c1e76137)
